src/core/lib/security/transport/server_auth_filter.cc

```diff
@@ -49,6 +49,8 @@ struct call_data {
   size_t num_consumed_md;
   grpc_closure cancel_closure;
   gpr_atm state;  // async_state
+  grpc_error* recv_trailing_metadata_error;
```

Comment:
Please move these up to line 47, so that all fields related to recv_trailing_metadata are in one place.